gce-jetty/src/main/docker/Dockerfile

```diff
@@ -27,11 +27,14 @@ RUN mv /usr/local/jetty-distribution-* $JETTY_HOME && chown -R jetty:jetty $JETT
 ENV JETTY_BASE /var/lib/jetty
 RUN mkdir -p $JETTY_BASE
 WORKDIR $JETTY_BASE
-RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,setuid && \
+RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,jsp,jstl,setuid && \
     chown -R jetty:jetty $JETTY_BASE && \
-    mkdir /tmp/jetty
+    mkdir /tmp/jetty && chown -R jetty:jetty /tmp/jetty
+
+RUN ln -s $JETTY_BASE/webapps/root /app
+WORKDIR /app
```

Comment:
Why make the workdir /app?
This is a deviation from the official jetty-docker image that may hurt portability of webapps (albeit badly written ones that assume working dir).